DigitalLearningSolutions.Data/DataServices/UserDataService/DelegateUserDataService.cs

```diff
@@ -125,6 +125,7 @@ public partial class UserDataService
                 da.UserID,
                 da.CentreSpecificDetailsLastChecked,
                 da.RegistrationConfirmationHash,
+                da.RegistrationConfirmationHashCreationDateTime,

```

Comment:
We need to revert the changes that bring the `RegistrationConfirmationHashCreationDateTime` into the view model since we no longer do anything with it.

Reply:
This is addressed.